src/NexusMods.App.UI/LeftMenu/Items/IconViewModel.cs

```diff
@@ -15,5 +15,5 @@ public class IconViewModel : AViewModel<IIconViewModel>, IIconViewModel
 
     [Reactive] public string[] Badges { get; set; } = [];
 
-    [Reactive] public ReactiveCommand<NavigationInformation, Unit> NavigateCommand { get; set; } = ReactiveCommand.Create<NavigationInformation>(_ => { }, Observable.Return(false));
+    [Reactive] public ReactiveCommand<NavigationInformation, Unit> NavigateCommand { get; set; } = ReactiveCommand.Create<NavigationInformation>(_ => { }, Observable.Return(true));
```

Comment:
Why did you change this?

Reply:
To give some context, I originally put the default value to `false` because anyone that wants to add navigation has to supply their own command anyways.

Reply:
Ah it was mostly for preview purposes, so I could test the hover states. I can revert it if you want but, if it doesn't hurt I would leave it like that for the preview.